frontend/packages/ocs-independent-plugin/src/plugin.ts

```diff
@@ -0,0 +1,120 @@
+import {
+  DashboardsTab,
+  ModelDefinition,
+  ModelFeatureFlag,
+  Plugin,
+  DashboardsCard,
+  ActionFeatureFlag,
+} from '@console/plugin-sdk';
+import { GridPosition } from '@console/shared/src/components/dashboard/DashboardGrid';
+import { OCSServiceModel } from '@console/ceph-storage-plugin/src/models';
+import { detectIndependentMode } from './feature';
+import { INDEPENDENT_FLAG } from './consts';
```

Comment:
I'd suggest keeping flag name constants inside the plugin entry module (this file).

Reply:
> I'd suggest keeping flag name constants inside the plugin entry module (this file).

Update: if the flag is managed via detector function, move it to corresponding `features` module. If it's shared between multiple packages, move it to `console-shared` constants.